src/test/java/edu/hm/hafner/analysis/parser/MsBuildParserTest.java

```diff
@@ -705,6 +705,32 @@ void shouldDetectKeywordsInRegexCaseInsensitive() {
         });
     }
 
+    /**
+     * Update regular expresion to detect logging prefixes like 17:4>.
```

Comment:
typo: expression

is there a JavaDoc warning because of the 17:4>? Maybe you need to wrap that part in a pre tag.